### DISCUSSION TITLE
Accessing the terminal in a node template

### DISCUSSION THREAD

**Discussion by danwoods** [codesandbox/codesandbox-client#5130] [Q&A]
I'm trying to create a sandbox using the node template but I'm running into issues accessing the terminal. I have a sandbox [here](https://codesandbox.io/s/0joix) that should be using a node template and have a defined start script. It shows a 504 and doesn't give me access to the terminal. What am I doing wrong?

**Comment by SaraVieira** [comment]
Hey

You are getting a 504 because your sandbox does not a "serve" option meaning the browser will not show anything so we show you a 504 and we should fix that because it is super confusing but there is nothing wrong, we just don't have anything to show in the browser.

The default terminal is read-only, to create a write terminal you can click here:

![CleanShot 2020-11-16 at 19 07 24](https://user-images.githubusercontent.com/1051509/99290775-0d880200-283f-11eb-888d-104ab6a19e1f.png)


Let me know if you can now create a new terminal

**Reply by danwoods** [reply]
Thanks for the feedback! I don't supposed there's a way to create a writable terminal when uploading through the define API, is there?

**Reply by SaraVieira** [reply]
ohhh, in that case, no because you need to be the owner of the sandbox to create these terminals and all the define sandboxes are anonymous, so you would have to fork the created one :(

**Reply by danwoods** [reply]
Thanks for the response 👍
